examples/feeds/create_feed_item_set.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Creates a new feed item set for a specified feed.
+
+The feed must belong to either a Google Ads location extension or an affiliate
+extension. This is equivalent to a "location group" in the Google Ads UI. See
+https://support.google.com/google-ads/answer/9288588 for more detail.
+"""
+
+import argparse
+import sys
+import uuid
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, feed_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_id: the ID for a Feed belonging to the given customer.
+    """
+    feed_item_set_service = client.get_service(
+        "FeedItemSetService", version="v6"
+    )
+    feed_item_set_operation = client.get_type(
+        "FeedItemSetOperation", version="v6"
+    )
+
+    feed_item_set = feed_item_set_operation.create
+    feed_resource_name = client.get_service(
+        "FeedService", version="v6"
+    ).feed_path(customer_id, feed_id)
+    feed_item_set.feed = feed_resource_name
+    feed_item_set.display_name = "Feed Item Set #%s" % uuid.uuid4()
+
+    """A feed item set can be created as a dynamic set by setting an optional
+    filter field below. If your feed is a location extension, uncomment the code
+    that sets dynamic_location_set_filter. If your feed is an affiliate
+    extension, set dynamic_affiliate_location_set_filter instead.
+    """
+    # 1) Location Extension
+    # NOTE: Does not support Curated location extensions.
```

Comment:
I tried to test this by creating a location extension using Curated locations, since I do not have a Google My Business account, and received error: "The feed type of the parent Feed is not compatible with the type of dynamic filter being set." According to the [docs](https://developers.google.com/google-ads/api/reference/rpc/v6/FeedItemSetErrorEnum.FeedItemSetError), this happens when the feed item set type isn't location-based, e.g. "you can only set dynamic_location_set_filter for LOCATION feed item sets." I would think this would work for location feeds from Google My Business but haven't found a way to verify (that doesn't involve creating a GMB account and plenty of mock data).

Reply:
Tested